src/reducers/columnStateHelper.js

```diff
@@ -0,0 +1,359 @@
+/**
+ * Copyright Schrodinger, LLC
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @providesModule columnStateHelper
+ */
+
+'use strict';
+
+import FixedDataTableWidthHelper from 'FixedDataTableWidthHelper';
+import React from 'React';
+import emptyFunction from 'emptyFunction';
+import invariant from 'invariant';
+import shallowEqual from 'shallowEqual';
+
+var EMPTY_OBJECT = {};
+var HEADER = 'header';
+var FOOTER = 'footer';
+var CELL = 'cell';
+var DRAG_SCROLL_SPEED = 15;
+var DRAG_SCROLL_BUFFER = 100;
+
+function _selectColumnElement(type: string, columns: Array): Array {
```

Comment:
Do we actually have anything that checks flowtype? I don't think its actually enabled in the repo..